### DISCUSSION TITLE
use in email-based patch workflow?

### DISCUSSION THREAD

**Discussion by alterae** [jj-vcs/jj#4679] [Q&A] [answered]
hi! i do a lot of my coding over on Sourcehut, which uses a patch workflow based on [`git send-email`](https://git-send-email.io) rather than GitHub-style pull requests. i was wondering what it would look like to use `jj` in such a workflow

**Comment by ilyagr** [comment, marked as answer]
Currently, it'd probably be easiest to have a `jj` repository [co-located with Git](https://martinvonz.github.io/jj/prerelease/git-compatibility/#co-located-jujutsugit-repos) and use `git send-email` itself. We don't have any explicit support for this workflow at the moment -- there is no `jj send-email` or equivalent -- but doing it via Git should work fine. I'm not sure we have many users that tried it; we appreciate any experience reports and bug reports.

It's an interesting question whether `jj` could have a better UI for it than something equivalent to just using Git as I described; I'm not sure.

**Update:** https://martinvonz.github.io/jj/latest/FAQ/#should-i-co-locate-my-repository might be quicker to read.

**Reply by ilyagr** [reply]
I've been reminded on Discord that this has been discussed on Discord. Here's one discussion I could find:

https://discord.com/channels/968932220549103686/968932220549103689/1273752874408476786

(The invite link is in our README). There might also be discussions here on GitHub. If not, this could become "the discussion", or you can open a feature request.

This also means that somebody other than me might have more thoughts and information.

**Reply by ilyagr** [reply]
A collection of Discord backlinks: https://discord.com/channels/968932220549103686/968932220549103689/1297615561047412827

(Though it might be nice if we started a discussion on GitHub for easier tracking of people's thoughts)

**Reply by alterae** [reply]
> A collection of Discord backlinks:

I saw! I think I actually prompted the conversation that lead to that collection. 

> (Though it might be nice if we started a discussion on GitHub for easier tracking of people's thoughts)

It very much would. More searchable and less ephemeral (see the other discord discussion immediately following those backlinks, in which we discuss the merits of putting important stuff in places that aren't discord).
